app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm.rb

```diff
@@ -97,6 +97,16 @@ def raw_reset
     raw_reboot_guest(:force => true)
   end
 
+  supports :terminate do
+    unsupported_reason_add(:terminate, unsupported_reason(:control)) unless supports_control?
+  end
+
+  def raw_destroy
+    raise "VM has no #{ui_lookup(:table => "ext_management_systems")}, unable to destroy VM" unless ext_management_system
+    with_provider_object(&:delete)
+    update!(:raw_power_state => "powering-down")
```

Comment:
The raw_power_state for VPC is "stopping" per the `POWER_STATES` map: https://github.com/ManageIQ/manageiq-providers-ibm_cloud/blob/master/app/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/vm.rb#L13

Reply:
The point of this is just to indicate immediately that something is changed before the refresh has run